kyverno/authorization/host-namespaces/disallow-host-network.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-host-network
+  annotations:
+    policies.kyverno.io/description: Host network Controls whether the pod may use the node network namespace.
```

Comment:
@igorkchyts  Would change to `Host network Controls whether the pod may use the host's network namespace. Providing access to the host's network gives the pod access to the loopback device. Pods with access to the loopback device can be used to snoop on network activity of other pods on the same node`.